CSS/quizes.css

```diff
@@ -0,0 +1,116 @@
+body{
+	background-color: var(--light-primary-color);
+}
+
+.list-inline-item a{
+	color: unset;
+	text-decoration: none;
+}
+
+.category-container{
+	position: relative;
+	/* width: 600px; */
+	margin-left: 30px;
+	margin-right: 30px;
+	margin-top: 50px;
+}
+
+.slider-container{
+	position: relative;
+}
+
+.slider{
+	align-items: center;
+}
+
+.slider::before{
+	display: none;
+}
+#prev1,#prev2{
+	position: absolute;
+	left: 30px;
+	top: 0;
+	height: 100%;
+    opacity: 0.7;
+    background: white;
```

Comment:
alignment issue. Use prettier to fix these issues

Reply:
fixed in develop branch